reference/test/matrix/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ ginkgo_create_test(csr_kernels)
 ginkgo_create_test(dense_kernels)
 ginkgo_create_test(diagonal_kernels)
 ginkgo_create_test(ell_kernels)
+ginkgo_create_test(fbcsr_kernels test_fbcsr_sample)
```

Comment:
This is the other place where the same functionality (of sample matrix generation) is needed. I thought it's fine to have dependencies for tests if it can avoid repeated compilation of the same code.